### DISCUSSION TITLE
History View with new entities

### DISCUSSION THREAD

**Discussion by Renegade605** [home-assistant/frontend#17155] [Dashboard]
In the history, if a certain sensor hasn't/didn't exist for the entire time scale requested, it should still match the time scale of the other sensors. ie. In the screenshot below, the time is 18:55 to present, but "Test Humidity Derivative" was created at 19:15. Before 19:15, it should show "unknown" or nothing at all instead of the state it was created with.
![image](https://github.com/home-assistant/frontend/assets/5857062/6da20e73-d1c1-437f-8ab2-6c0a4cec2ac4)
![image](https://github.com/home-assistant/frontend/assets/5857062/930de9fc-fd89-4d8d-ab2d-c62eb5fbfd3c)

**Comment by ildar170975** [comment]
Check https://github.com/home-assistant/frontend/issues/16660
